### PR TITLE
Fixes cables on FBPs (and probably cyborgs)

### PR DESCRIPTION
**Pull request by MisterLayne**
Fixes the thing where if you healed a cyborg with cables who was already full health, it would eat the cables anyways.

### CODE REVIEW

**Comment by Anewbe** [comment]
Did you test this?

**Comment by MisterLayne** [comment]
Tested and approved.

File: code/modules/surgery/robotics.dm [hunk 1]
```diff
@@ -168,10 +168,16 @@
 			var/obj/item/organ/external/affected = target.get_organ(target_zone)
 			if(istype(tool,/obj/item/stack/cable_coil/))
 				var/obj/item/stack/cable_coil/C = tool
-				if(!C.can_use(5))
-					user << "<span class='danger'>You need ten or more cable pieces to repair this damage.</span>" //usage amount made more consistent with regular cable repair
-					return SURGERY_FAILURE
-				C.use(5)
+				if(affected.burn_dam == 0)
+					to_chat(user, "<span class='notice'>There are no burnt wires here!</span>")
+					return
```
Comment (Anewbe) [inline code/modules/surgery/robotics.dm]: return SURGERY_FAILURE here, I think.

File: code/modules/surgery/robotics.dm [hunk 2]
```diff
@@ -168,10 +168,16 @@
...
+				else
+					if(!C.can_use(5))
+						to_chat(user, "<span class='danger'>You need ten or more cable pieces to repair this damage.</span>") //usage amount made more consistent with regular cable repair
```
Comment (Anewbe) [inline code/modules/surgery/robotics.dm]: I don't think it's ten anymore, mind changing the line to five or more?

**Pull request merged**